.jenkins/Jenkinsfile

```diff
@@ -0,0 +1,93 @@
+//The Jenkinsfile runs entire build in a golang container and mounts the /var/run/docker.sock file to allow access to the host docker within the container
+node('wcm') {
+    def build_ok = true
+    docker.image('golang:1.15-buster').inside('-u root --net=host -v /var/run/docker.sock:/var/run/docker.sock') {
+        environment{
+            CGO_ENABLED = 0
+            GO111MODULE = on
+        }
+        stage ('Install Docker Client') {
```

Comment:
I refactored what was in nsm-nse to use the same base for our runner container we use in circleci to avoid having to install all these type of dependencies--docker client, kubectl, helm, kind, go, git, etc.  I'm looking to get a more standard set of base container images curated by the SRE team that we can use as our jenkins agent setting.  We don't have to change this now but it'll be something we want to follow-up on.

See here:  https://github.com/cisco-app-networking/nsm-nse/blob/devop_jenkins/.jenkins/Jenkinsfile#L14

Reply:
ok thanks, good to know.